tests/codegen/fcode/scripts/numpyext.py

```diff
@@ -47,8 +47,8 @@
 print(np_sum(x17)==n*m*2)
 
 x18 = zeros((3, 3))
-x19 = diag(x18)
-x20 = diag(x18)
+#x19 = diag(x18)
+#x20 = diag(x18)
```

Comment:
Was this code working before? Did we support `diag` at all?

By the way, we should comment out the import statement for `diag` as well.

Reply:
it was never tested so I'm not sure